packages/jest-circus/src/run.ts

```diff
@@ -91,12 +112,22 @@ const _runTestsForDescribeBlock = async (
       }
       case 'test': {
         const hasErrorsBeforeTestRun = child.errors.length > 0;
+        const hasRetryTimes = retryTimes > 0;
         await _runTest(child, isSkipped);
 
+        //If immediate retry is set, we retry the test immediately after the first run
         if (
+          retryImmediately &&
```

Comment:
What if the option would be named `retryImmediately`?

Reply:
Sorry, don't get it. If I passed `retryImmediately` it wouldn't work.

Reply:
Sorry, I did not explain well. The comment is not about this particular line of code.

While looking through and I really liked how well `retryImmediately` reads. So I thought: what if the `immediately` option would be actually named `retryImmediately`.

Reply:
Ok, I also thought about it. Can agree with you, I just got `immediately` property name from the origin issue.  `retryImmediately` is more clear.

Reply:
I renamed the property